client/components/select-dropdown/docs/example.jsx

```diff
@@ -43,10 +43,10 @@ var SelectDropdownDemo = React.createClass( {
 	},
 
 	render: function() {
-		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		let toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
```

Comment:
I wonder why I don't get the eslint warning

![image](https://cloud.githubusercontent.com/assets/77539/17602139/ab8c6ea8-5fe1-11e6-8aff-03420edc131e.png)